### ISSUE TITLE
Test suite failed to run

### ISSUE THREAD

**Issue by hyochan** [th3rdwave/react-native-safe-area-context#48]
```
Test suite failed to run

    TypeError: Cannot read property 'Constants' of undefined

      at Object.RNCSafeAreaViewConfig (node_modules/react-native-safe-area-context/lib/commonjs/InitialWi
ndowSafeAreaInsets.ts:9:17)
      at Object.<anonymous> (node_modules/react-native-safe-area-context/lib/commonjs/index.tsx:6:1)

Test Suites: 1 failed, 5 passed, 6 total
```

I am currently suffering from the above jest test after upgrading to react-navigation v5.
Here is the [pr of the repo](https://github.com/dooboolab/dooboo-native-ts/compare/enh/navigation?expand=1).

Here is the error in [circleci](https://circleci.com/gh/dooboolab/dooboo-native-ts/1168?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link).

**Comment by TuyenNguyenAcazia** [comment]
same problem

**Comment by janicduplessis** [comment]
Sorry about that, fixed in 0.7.1
